helm/panda/charts/jedi/templates/deployment.yaml

```diff
@@ -76,6 +76,9 @@ spec:
           command: ["/bin/sh", "-c"]
           args:
             - until  /usr/pgsql-*/bin/pg_isready -h ${PANDA_DB_HOST} -p 5432; do echo waiting for database; sleep 2; done;
+              chmod -R 777 /var/log/panda;
```

Comment:
The same comment as ActiveMQ container. It is done in https://github.com/PanDAWMS/panda-k8s/blob/main/helm/panda/charts/jedi/sandbox/init-jedi#L17 anway.
Also you don't have to delete pid and lock files since they are created in container instead of PV.